README.md

```diff
@@ -262,6 +262,35 @@ The focus of Phase 2 will be to Verify the state from layer 1 and implement the
 
 </details>
 
+### Phase 3: Starknet decentralization begins 🚧
+
+<details>
+<summary></summary>
+
+Juno can synchronize Starknet state from other full nodes with the aim of decentralizing Starknet by removing the dependency from the centralized sequencer.
+
+
+Snap sync is implemented, significantly reducing sync times and the implementation of JSON RPC’s Trace API will allow users to better understand its execution.
```

Comment:
Just realized: we already implement the trace API. Can we remove that clause from this sentence?

Reply:
Updated

Reply:
Thanks. I think it would be better if we removed the part about the trace API. I am concerned readers may interpret it as saying that the trace API isn't implemented.